.github/workflows/automate-review-labels.yml

```
@@ -39,24 +40,35 @@
                   let text = context.payload.comment.body;
                   const regex = new RegExp(team + '[^\-]|' + team + '$');
                   let result = regex.test(text);
+                  let found_label = false;
                   if (result) {
                       label = team.replace("@conda-forge/help-", "");
                       label_no_staged = label.replace("@conda-forge/", "");
+                      let found_label = true;
                       github.rest.issues.addLabels({
                           issue_number: context.issue.number,
                           owner: context.repo.owner,
                           repo: context.repo.repo,
                           labels: [label_no_staged, 'review-requested']
-                      })
-                      github.rest.issues.removeLabel({
-                          issue_number: context.issue.number,
-                          owner: context.repo.owner,
-                          repo: context.repo.repo,
-                          name: ['Awaiting author contribution']
                       });
                       // TODO: Maybe use GitHub API to request review / assign team
                   }
-              }
+              };
+              return found_label
+        - name: add-labels
+          if: >
+            steps.set-result.outputs.result
```

I'm not sure if the `True` is actually evaluated like that. I will test that in the test repo.

---

```suggestion
            steps.check-teams.outputs.result
```
You need to use the name?/id? field of the step you want output from.

---

Whops forgot that, thanks.